src/accounts/AccountPage.tsx

```diff
@@ -210,9 +207,7 @@ const AccountPage: FC = () => {
     <Page titleTag={pageTitleSuffixer(['Account Settings Page'])}>
       <AccountHeader testID="account-page--header" />
       <UsersProvider>
-        <UserAccountProvider>
-          <AccountAboutPage />
-        </UserAccountProvider>
+        <AccountAboutPage />
```

Comment:
Same comment here. AFAIK, this is a shared component. If we delete this provider, and aren't in the cloud, this component will no longer have access to the `UserAccountProvider`. So we need to determine whether or not that component needs access to the provider in CLOUD v. non-CLOUD.